lib/puppet/provider/base_dsc/powershell.rb

```diff
@@ -60,5 +60,10 @@ def ps_script_content(mode)
     template.result(binding)
   end
 
-
+  def ps_invoke_content(mode)
```

Comment:
It wasn't clear to me whether we reached consensus to keep `ps_script_content` or not. If that's the case, then I'll change back to that